build.gradle

```diff
@@ -68,3 +80,29 @@ tasks.withType(JavaCompile) {
     options.encoding = 'UTF-8'
     options.annotationProcessorGeneratedSourcesDirectory = file("src/generated/java")
 }
+
+tasks.named('test') {
+    useJUnitPlatform()
+}
+
+
+configurations {
+    cucumberRuntime {
+        extendsFrom testImplementation
+    }
+}
+
+task cucumberCli() {
+    dependsOn assemble, testClasses
+    doLast {
+        javaexec {
+            main = "io.cucumber.core.cli.Main"
+            classpath = configurations.cucumberRuntime + sourceSets.main.output + sourceSets.test.output
+            args = [
+                    '--plugin', 'pretty',
+                    '--plugin', 'html:target/cucumber-report.html',
+                    '--glue', 'org.mifos.connector.slcb.cucumber',
```

Comment:
wrong package name, `org.mifos.connector.slcb.cucumber` doesn't make sense to me.